galaxyproject/galaxy
# Spanish Translations for remaining untranslated UI components

We have a fantastic @galaxyproject/training-spanish team which have done lots of fantastic work translating (and hopefully continuing to translate UI components in the future.

This PR could use additional review to make sure I'm doing things with JS best practices since I speak more spanish than JS (hah).

All of the translations were done by the GTÑ team here https://docs.google.com/document/d/1cjv_I3toCa11aLbyC5pv17UFID3TH9D7UqTvFCthlCk/edit

Fixes https://github.com/galaxyproject/galaxy/issues/11982

## How to test the changes?
(Select all options that apply)
  1. Change to spanish
  2. Check that the only english is in the toolpanel

## License
